mdx/guides/rfqt-in-the-0x-api.mdx

```diff
@@ -0,0 +1,164 @@
+
+# Dedicated Makers
+
+In addition to the 0x API configuration identifying trusted takers, it also contains a list of specific market makers that participate in the RFQ-T model. Each maker is identified by an HTTP endpoint URL, and each endpoint has an associated list of asset pairs for which that endpoint will provide quotes.
+
+For the instance at `api.0x.org`, the 0x team is maintaining a list of trusted market makers, which at this time includes 0x's own Periscope Trading, as well as a handful of other partners.
+
+Whenever a trusted taker requests a quote for an asset pair that's served by a known maker endpoint, the 0x API will solicit that maker for a quote on behalf of the taker, and will consider that quote during its aggregation of various liquidity sources.
+
+# Maintaining Trust
+
+RFQ-T market makers are never obligated to provide a quote. If they cannot provide a competitive price at a given moment, for example, they may decline to make an offer. Moreover, they may decline to respond for any reason at all.
+
+Because the taker address is always known to the market maker, the maker can easily monitor taker behavior over time. Makers can potentially identify arbitrageurs, by identifying specific taker addresses that seem to repeatedly take advantage of the maker's mispriced orders. And because the maker has the option to choose whether or not to respond to the quote request, they can easily blacklist taker addresses that they deem to be arbitrageurs.
+
+Furthermore, the request sent from the 0x API to an RFQ-T market maker also includes the API key associated with the trusted integration client. In the same way that makers can blacklist taker addresses, they also have the option to blacklist specific applications that integrate with the 0x API.
+
+In order to retain access to RFQ-T liquidity, it's important for clients of the 0x API to be good citizens of the marketplace. For end users of trusted integrations, being a good citizen means actually filling the orders they claim to intend to fill. And for trusted integrations, it means properly managing their users' intentions; more specifically, it means adhering to the flow of using indicative quotes by default, and firm quotes only when appropriate.
+
+# Quote Types: Indicative vs Firm
+
+In order to improve capital efficiency, the RFQ-T model introduces a new request/response flow: first request pricing, then request a fillable quote.
+
+When a market maker provides a fillable quote, which can be settled by the taker's submitting it to the 0x protocol, the taker has an expectation that the swap will actually settle, that it won't revert due to a lack of available maker assets. To uphold this expectation, makers will refrain from using those same assets as the basis for any other quotes, until the quote's expiry.
```

Comment:
I don't think all market makers are doing this exactly as described, they are likely taking an educated guess on the likelihood that the user will actually fill, and not 100% reserving funds for every firm quote they give out.